### PR TITLE
1114 in collection

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1114 .

### CODE REVIEW

File: src/main/resources/alma/common/fields.xml [hunk 1]
```diff
@@ -751,5 +751,68 @@
...
 
+  <!-- TODO: inCollection: missing edoweb, rpb, publisso, also type-element -->
+  <entity name="inCollection[]" flushWith="record">
+    <!-- ZDB -->
+    <entity name="" sameEntity="true" reset="true">
+      <data name="label" source="@zdbId">
+        <constant value="Zeitschriftendatenbank (ZDB)" />
+      </data>
+      <data name="id" source="@zdbId">
+        <constant value="http://www.lobid.org/resources/HT014846970#!" />
```
Comment (dr0i) [inline src/main/resources/alma/common/fields.xml:762]: Use the generic URL without 'www'.

File: src/test/resources/alma/HT005207972.json [hunk 2]
```diff
@@ -115,7 +115,7 @@
     },
     "resultOf" : {
       "type" : [ "CreateAction" ],
-      "endTime" : "2021-02-15T08:30:45",
+      "endTime" : "2021-02-15T12:54:38",
```
Comment (dr0i) [inline src/test/resources/alma/HT005207972.json:118]: Omit the adding of this file as discussed.

File: src/main/resources/alma/common/fields.xml [hunk 3]
```diff
@@ -751,5 +751,68 @@
...
+    </entity>
+    <!-- nwbib -->
+    <entity name="" sameEntity="true" reset="true">
+      <data name="label" source="084  .2">
+        <equals string="nwbib" />
+        <constant value="Nordrhein-Westfälische Bibliographie (NWBib)" />
+      </data>
+      <data name="id" source="084  .2">
+        <equals string="EZB" />
+        <constant value="http://www.lobid.org/resources/HT014176012#!" />
```
Comment (dr0i) [inline src/main/resources/alma/common/fields.xml:794]: This collection ("...HT014176012#!" ) seems not be tested anywhere.

File: src/test/resources/alma/HT012734884.json [hunk 4]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (dr0i) [inline src/test/resources/alma/HT012734884.json:1]: Not sure why adding this file. What does it test what other files not already test?

**Review by dr0i — CHANGES_REQUESTED** [review]
As discussed offline - don't commit-message "run mvn".
Better add some more infos like "Add collections "Zeitschriftendatenbank (ZDB)",  "Elektronische Zeitschriftenbibliothek (EZB)" and "eResource package".
We then could easily search in the future the now missing collections, e.g. "edoweb".

File: src/main/resources/alma/common/fields.xml (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/fields.xml:762]: Did that with the new last commit.

File: src/test/resources/alma/HT012734884.json (hunk 4 above)
Comment (TobiasNx) [reply src/test/resources/alma/HT012734884.json:1]: Thought I should only, or all files that change concerning the change in the morph. Will reduce the number of testfiles in the future.

File: src/main/resources/alma/common/fields.xml (hunk 3 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/fields.xml]: There was an error with the nwbib mapping, this is fixed now.

File: src/test/resources/alma/HT005207972.json (hunk 2 above)
Comment (TobiasNx) [reply src/test/resources/alma/HT005207972.json]: did that.

File: src/main/resources/alma/common/fields.xml (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/fields.xml]: fixed this with the last commit

**Comment by TobiasNx** [comment]
> As discussed offline - don't commit-message "run mvn".
> Better add some more infos like "Add collections "Zeitschriftendatenbank (ZDB)", "Elektronische Zeitschriftenbibliothek (EZB)" and "eResource package".
> We then could easily search in the future the now missing collections, e.g. "edoweb".

Changed the commit msg to more instructive versions.

File: src/test/resources/alma/HT012734884.json (hunk 4 above)
Comment (dr0i) [reply src/test/resources/alma/HT012734884.json:1]: I don't understand what you mean by this.
If this resource doesn't test anything new it should not be commited.

File: src/test/resources/alma/HT017131400.json [hunk 5]
```diff
@@ -0,0 +1,122 @@
+{
```
Comment (dr0i) [inline src/test/resources/alma/HT017131400.json:1]: This resource doesn't test 'inCollection'. It should be removed.

**Review by dr0i — CHANGES_REQUESTED** [review]
There are still two resources as test files which doesn't seem to be needed.

File: src/test/resources/alma/HT012734884.json (hunk 4 above)
Comment (dr0i) [reply src/test/resources/alma/HT012734884.json:1]: +1

File: src/main/resources/alma/common/fields.xml (hunk 3 above)
Comment (dr0i) [reply src/main/resources/alma/common/fields.xml]: +1

File: src/test/resources/alma/HT005207972.json (hunk 2 above)
Comment (dr0i) [reply src/test/resources/alma/HT005207972.json]: +1

File: src/main/resources/alma/common/fields.xml (hunk 1 above)
Comment (dr0i) [reply src/main/resources/alma/common/fields.xml]: Hm, there are still many links starting with 'www.lobid.org' left.

Comment (dr0i) [reply src/main/resources/alma/common/fields.xml]: +1

File: src/test/resources/alma/HT012734884.json (hunk 4 above)
Comment (dr0i) [reply src/test/resources/alma/HT012734884.json:1]: Discussed offline: this resource checks multiple inCollection eResources, so this is ok.

File: src/test/resources/alma/HT017131400.json (hunk 5 above)
Comment (dr0i) [reply src/test/resources/alma/HT017131400.json]: The resource is removed, but it still exists in the tar.bz2. We keep it there nonetheless, bacause we will need it in the near.

**Review by dr0i — APPROVED** [review]

**Pull request merged**